README.md

```diff
@@ -8,7 +8,19 @@ Documentation can be found at <https://documentation.onfido.com>
 
 ## Installation
 
-_Coming soon_
+For Maven projects, add the below to your pom file:
+```xml
+<dependency>
+      <groupId>com.onfido</groupId>
+      <artifactId>onfido-api-java</artifactId>
+      <version>1.0.0</version>
```

Comment:
Since these are instructions, maybe we should specify to use the latest version instead, so that we don't have to update this readme every time we deploy a new version. Or, at the very least add a comment saying that this is an example and to use the latest version available.